hadoop-tools/hadoop-msgraph/src/test/resources/msgraph_api_response1.txt

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
rename as .json

you need to tell something (RAT?) to ignore the fact these files have no licence